mail_tracking_mailgun/models/res_partner.py

```diff
@@ -25,10 +25,13 @@ def _email_bounced_set(self, reason, event):
         for partner in self:
             if not partner.email:
                 continue
-            body = _('Email has been bounced: %s\n'
-                     'Reason: %s\n'
-                     'Event: %s') % (partner.email, reason,
-                                     event and event.id or _('unknown'))
+            body = _(
+                'Email has been bounced: %s\n'
+                'Reason: %s\n'
+                'Event: '
+                '<a href=# data-oe-model=mail.tracking.event data-oe-id=%d>'
```

Comment:
```suggestion
                '<a href="#" data-oe-model="mail.tracking.event" data-oe-id="%d">'
```

Reply:
Applied in #434, but I'll amend with the changes made ther to avoid translation change